addons/udes_stock/views/stock_picking.xml

```diff
@@ -289,4 +289,23 @@
         </field>
     </record>
 
+    <record id="view_picking_warning" model="ir.ui.view">
+        <field name="name">stock.inventory.adjust.reserved.view.form</field>
+        <field name="model">stock.inventory</field>
+        <field name="priority">20</field>
+        <field name="arch" type="xml">
+            <form string="Warning">
+                <group>
+                    <p>
+                        Do you wish to continue?
+                    </p>
+                </group>
+                <footer>
+                    <button name="button_validate_continue" string="_Continue" type="object" class="btn-primary"/>
```

Comment:
Why `_Continue` (with underscore at the start)? Can see this is done elsewhere in `udes_stock` but can't find other instances of it. Is it to do with translation?

Reply:
Update: Found a few examples in Odoo modules (`_Apply`, `_Import` etc). Assume it is translation related, but discussion is out of scope for this story anyway. :)